### PR TITLE
fixes and new translations

### PR DESCRIPTION
**Pull request by Gralnd**

### CODE REVIEW

File: pages/post/createpost.vue
Comment (ms0ur) [inline pages/post/createpost.vue:1]: rename back

**Review by ms0ur — CHANGES_REQUESTED** [review]
Rename new post page back to pages/post/new.vue

File: pages/post/createpost.vue
Comment (Gralnd) [reply pages/post/createpost.vue:1]: nuh

Comment (ms0ur) [reply pages/post/createpost.vue:1]: then i close pr without merging

**Review by ms0ur — APPROVED** [review]

**Pull request merged**